rule_packages/c/IntegerOverflow.json

```diff
@@ -0,0 +1,124 @@
+{
+  "CERT-C": {
+    "INT30-C": {
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "Unsigned integer expressions do not strictly overflow, but instead wrap around in a modular way. If the size of the type is not sufficient, this can happen unexpectedly.",
+          "kind": "problem",
+          "name": "Ensure that unsigned integer operations do not wrap",
+          "precision": "high",
```

Comment:
This point is subjective, but a lot of the results I am seeing on openpilot won't wrap because the operands are bound in some scope that is not part of the analysis, such as a different function. Interprocedural data-flow to implement more precise range analysis like the `getStatedValue` predicate could help, but it would add a lot of complexity to the implementation and worsen performance.

I think the heuristic as it is implemented now is good enough but nonetheless noisy, so I would lower the query's precision to medium.

Reply:
```suggestion
          "precision": "medium",
```

Reply:
This also applies to `INT32-C`.